src/Scripts/Cli/Command/RunCommand.php

```diff
@@ -47,12 +47,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->scriptRunner->setWorkingDir($this->getWorkingDir());
 
         $scriptName = $this->io->getArgument('script');
-        $result = $this->scriptRunner->run($scriptName);
-
-        if (!$result) {
-            $this->io->error(sprintf('Unable to find script "%s"', $scriptName));
-
-            return 1;
-        }
```

Comment:
Howcome we lost this?

Reply:
Seems to be a hangover from an old implementation from looking through the git history. When this commit was implemented 10efb322811f608b83542cda373e405fd1f60f7b (not sure if github will help me out here with that hash) – we removed the idea of the script runner returning a status code. So this to me looked like a bug that we were still checking it.

Reply:
Yeah that seems to make sense - https://github.com/jadu/meteor/blob/6a2c7302e6f2cc255b9afdece2011f2c68099f3c/src/Scripts/ScriptRunner.php#L63-L70